src/envs/react/theme-with-centering.tsx

```diff
@@ -1,6 +1,6 @@
 export {};
 
-// TODO - 
+// TODO -
```

Comment:
careful, this could modify all the components in the project!

Reply:
It's the prettier, I removed this change.